pywb/static/wombat.js

```diff
@@ -1110,6 +1111,12 @@ var _WBWombat = function($wbwindow, wbinfo) {
                     value = rewrite_inline_style(value);
 
                 } else if (should_rewrite_attr(this.tagName, lowername)) {
+                    if (this.tagName === 'SCRIPT' && !this.__$removedWBOSRC$__) {
```

Comment:
Can this logic be moved to its own function, and then just have a` _check_wbosrc(elem)` since its repeated 3 times? Will make maintenance easier. And then should be good!

Reply:
Done